tests/mocked_tests/test_backup.py

```diff
@@ -1,19 +1,21 @@
 import json
 from pathlib import Path
+
 from openshift_resources.velero.backup import Backup
 
 path = str(Path(__file__).parent)
 
 
 def test_backup():
     # import sample Backup CR json
-    with open(path + "/sample_output/backup.json", "r") as backup_json:
+    with open(path + "/sample_output/backup_wo_api_version.json", "r") as backup_json:
         backup_json_str = backup_json.read()
         backup_dict = json.loads(backup_json_str)
 
-        # "deserialize" dict to Backup object
-        b = Backup(**backup_dict)
-
-        # "serialize" Backup object to dict
-        b.dict()
+        br = Backup.build_with_default_client(**backup_dict)
 
+        # "deserialize" dict to Backup object
+        br.dict()
+        br.delete()
```

Comment:
how does this act if resource was not found?

Reply:
it throws an exception

Reply:
fixed